src/stan/mcmc/hmc/base_hmc.hpp

```diff
@@ -34,7 +34,7 @@ namespace stan {
           epsilon_jitter_(0.0) {}
```

Comment:
Yes, but it shouldn't hurt too much here. I think git will handle the merge fine.